uploader/push_to_sling.py

```diff
@@ -4,28 +4,60 @@
 import argparse
 import getpass
 import logging
-from pathlib import Path
+import yaml
 from pathlib import PurePath
 from datetime import datetime
 
+DEFAULT_SERVER = 'http://localhost:8080'
+DEFAULT_REPOSITORY = getpass.getuser() + '_' + datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
```

Comment:
I would leave the default repository as just the user name (without the current date component). That way it's ensured that they are working on their own local repository every time and not create spurious ones.

Reply:
The reason that I added the date component was in case we have 2 testing users that happen to have the same username. Remember that we are grabbing the local machine's user, not the kerberos ID, so we could easily have 2 people working in a "jon" repository (for example). If you are trying to avoid repository soup, perhaps we could come up some other pseudo-unique naming convention - maybe something including the machine hostname? Still not perfect, but better.

Reply:
Ah. As long as we understand this is not the full solution I'm good. The only problem is that should a test user want to change a few lines on a module and re-upload, they will get a completely new namespace.

Reply:
Yeah, the long-term plan is to have the repo identified permanently in the pantheon2.cfg file. I'll change the default repo name to username_hostname, and the user will still have the option to specify a different one if they so choose.

Reply:
:+1: